core/Symbols.cc

```diff
@@ -425,7 +425,9 @@ string ClassOrModuleRef::show(const GlobalState &gs) const {
     if (sym->name == core::Names::Constants::AttachedClass()) {
         auto attached = sym->owner.asClassOrModuleRef().data(gs)->attachedClass(gs);
         ENFORCE(attached.exists());
-        return fmt::format("T.attached_class (of {})", attached.show(gs));
+        // TODO: This change is needed for RBI generation.
+        return "T.attached_class";
+        // return fmt::format("T.attached_class (of {})", attached.show(gs));
```

Comment:
This makes error messages substantially more confusing. We should make separate printing methods before landing this so that we don't have to regress this error message.

Reply:
I removed this case, as the only one that fires is on `TypeMemberRef`.